[rq] Initial plan and tests

MindNumbing opened the pull request:
Signed-off-by: Steven Marshall <SMarshall@chef.io>

Commit:
[rq] Initial plan and tests

Signed-off-by: Steven Marshall <SMarshall@chef.io>

gavindidrichsen commented on controls/rq_works.rb:
```diff
@@ -0,0 +1,36 @@
+    its('exit_status') { should eq 0 }
+  end
+
+  rq_works = command("/bin/rq --version")
```
Please add the full /hab/pkgs/ORIGIN/rq/VERSION/TIMESTAMP/bin/rq path.

Commit:
[rq] Various refactors of existing code

Added inspec.lock to .gitignore
Updated README.md to latest binary format
Added plan_name to attributes.yml
Updates controls to add more prescriptive version checking.
Updated inspec.yml to latest version

Signed-off-by: Steven Marshall <SMarshall@chef.io>

gavindidrichsen reviewed (approved):
LGTM
